cherrytrack/blueprints/plates.py

```diff
@@ -10,6 +10,7 @@
 logger = logging.getLogger(__name__)
 
 bp = Blueprint("plates", __name__)
+cors = CORS(bp)
```

Comment:
Maybe because we need other extra information from cherrytrack not currently accessed with lighthouse?
In that case, can we still hide cherrytrack behind lighthouse?

Create a new endpoint in lighthouse that will proxy to cherrytrack. Happy to discuss it.

The reason is: cherrytrack is small app that handles permissions with api key. A frontend app like lighthouse-ui would need to have the api key in code *which it won't be correct* if we go to cherrytrack URL directly without any proxy in between to add the key or any authentication service to validate the cookie (or token).